client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
+import {
+  Button,
```

Comment:
I would think as many as you can fit until you hit the line width limit (which we should change to be 120 to match GH PR review width) and then you start breaking them out?